plugins/pi4j-plugin-ffm/src/test/resources/gpio-setup.sh

```diff
@@ -0,0 +1,22 @@
+#!/bin/sh
+
+modprobe configfs
```

Comment:
Why do we need this? Is this a new constraint to use FFM that we need to configure the kernel's GPIO before use? This might be quite an issue.

@FDelporte

Reply:
So scripts are just preparing testing environment (setup and clean `gpio-sim` and `i2c-stub`)

Reply:
Ok, good